sklearn/tree/_criterion.pyx

```diff
@@ -196,11 +197,10 @@ cdef class Criterion:
         cdef double impurity_right
 
         self.children_impurity(&impurity_left, &impurity_right)
-
         return ((self.weighted_n_node_samples / self.weighted_n_samples) *
-                (impurity - (self.weighted_n_right / 
+                (impurity - (self.weighted_n_right /
```

Comment:
Could you avoid modifying lines that are not related to the PR please? It works in your favor by speeding up the review as the diff is small...